frontend/packages/console-app/src/components/tour/tour-context.ts

```diff
@@ -39,7 +46,7 @@ export const tourReducer = (state: TourState, action: TourActions) => {
   }
 };
 
-type TourReducer = Reducer<TourState, TourActions>;
+type TourReducer = Reducer<TourState, { type: TourActions; payload?: boolean }>;
```

Comment:
Probably introduce a new for action say
```
type TourStateAction = { type: TourActions; payload?: { completed?: boolean }}
```
```suggestion
type TourReducer = Reducer<TourState, TourStateAction>;
```
would be better to make payload an object and with a `completed` prop. So it's more readable and we know what we are sending in the payload. In future we might need to send some other data inside payload.

Reply:
done